module.json

```diff
@@ -35,7 +35,12 @@
     "ualloc": "^1.0.0",
     "minar": "^1.0.0",
     "core-util": "^1.0.0",
-    "compiler-polyfill": "^1.1.0"
+    "compiler-polyfill": "^1.1.0",
+    "greentea-client": "~0.1.4"
+  },
+  "testDependencies": {
+    "unity": "^2.0.0",
+    "greentea-client": "~0.1.4"
```

Comment:
`greentea-client` is already listed as an unconditional dependency, so it doesn't need to be listed as a test dependency too.